doc/source/guide/FAQ.rst

```diff
@@ -106,7 +106,34 @@ To troubleshoot your Lux installation, we recommend cloning `this repo <https://
 
 The Lux Jupyter widget does not show up when I print a dataframe.
 """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
-  - If you recieve the output message :code:`LuxWidget(...)` but you do not see Lux widget show up, it is possible that the widget is not installed correctly. Run :code:`jupyter nbextension list` on the terminal, and you should see the following as one of the listed items. 
+To check if luxwidget is enabled, run the following code in your Jupyter Notebook/Lab:
+  
```

Comment:
We recommend using a simple toy example to test if your widget is showing correctly: 

```python
import lux
import pandas as pd
df = pd.read_csv("https://raw.githubusercontent.com/lux-org/lux-datasets/master/data/college.csv")
df
```